specs/posix/smart_drive_information.table

```diff
@@ -0,0 +1,33 @@
+table_name("smart_drive_information")
```

Comment:
What are your thoughts on something like `smart_info`? or `smart_drive_info`?

Reply:
`smart_drive_info` sounds good to me.